backend/Cargo.toml

```diff
@@ -34,10 +35,9 @@ wasm-bindgen = "0.2.87"
 wasm-bindgen-futures = "0.4.42"
 web-sys = { version = "0.3.64", features = ["console"] }
 web-time = "1.1.0"
-miniz_oxide = "0.7.4"
-itertools = "0.13.0"
 
 [target.'cfg(not(target_arch = "wasm32"))'.dependencies]
+geomedea = { git = "https://github.com/michaelkirk/geomedea" }
```

Comment:
btw this should compile in wasm now.

Reply:
In WASM, I want to disable the `writer` feature, but otherwise enable it. I fiddled around with specifying features based on architecture and landed here as something that works, but I'll take another look if there's a way to be more clear here...